doc/admin.md

```diff
@@ -162,11 +162,12 @@ support nulls or multi-attribute requests if they are encountered.
 #### `stats`
 
 It configures the periodic collection of statistics. Use `interval` in milliseconds to set the time between stats
-writings.
+writings. The `persistence` flag stores stats in the mongo backend.
```

Comment:
It would be great to provide more detail on this. For instance:

* In which collection at mongo backend are the stas persisted?
* How are the documents in that collection? Which fields they have? An example would be also great

Reply:
fixed